telegram_export/formatters/htmlformatter.py

```diff
@@ -4,6 +4,9 @@
 """
 from . import BaseFormatter
 
+import utils as export_utils
```

Comment:
```suggestion
from .. import utils as export_utils
```

Keeping it relative helps running the code as `python -m telegram_export` and does not show errors in IDEs like PyCharm.